vizro-core/docs/pages/user_guides/dashboard.md

```diff
@@ -184,4 +185,66 @@ To create a dashboard, do the following steps:
             Vizro().build(dashboard).run()
         ```
 
+
+## Adding a dashboard title
+
+By providing a `title` to the [`Dashboard`][vizro.models.Dashboard], a title will be added to each [`Page`][vizro.models.Page]
+on the top of the left navigation panel.
+
+!!! example "Adding a dashboard title"
```

Comment:
I would probably kill the example, it is just a bit overkill for what it explains